index.html

```diff
@@ -0,0 +1,156 @@
+    </article>
+    <aside class="transformer">
+      <img class="dvd" src="img/transformers_box_art.png" alt="Transformers DVD">
+      <img class="transformers_shadow" src="img/transformers_shadow.png" alt="transfomers box art shadow">
+      <h2 class="transformer_box_margin"><span>TRANSFORMERS</span> REVENGE OF THE FALLEN</h2>
+      <section class="transformer_box_padding review">
+        <p>A noisy, underplotted, and overlong special effects extravaganza that lacks a human touch.</p>
+      </section>
+      <section class="transformer_box_padding rotten_tomato">
+        <div class="tomato_headline">
+          <h2>ROTTEN TOMATO <span>RATING</span></h2>
+        </div>
+        <p class="tomato_percent">20%</p>
+        <progress class="progress" value="20" max="100">20%</progress>
+      </section>
+      <section class="transformer_box_padding premiere">
+        <div class="premiere_title">
+          <h3>WORLD PREMIERE</h3>
+        </div>
+        <p class="premiere_theater">DEN HAAG THEATRE</p>
+        <p class="premiere_country">THE NETHERLANDS</p>
+        <p class="premiere_date">JUNE 23, 2011</p>
+      </section>
+    </aside>
+    <article class="overflow">
+      <h2>OVERFLOW</h2>
+      <img class="water" src="img/water_crop.png">
+      <img class="overflow_bottom_text" src="img/overflow_bottom.png">
+      <p class="overflow_text text_one">On the night of Aug 8, 1975, a line of people frantically piled sandbags atop Henan Province's <strong>Banqiao Dam</strong> while being battered by the worst storm ever recorded in the region. They were in a race with the rapidly rising Ru River to save the dam and the millions of pople that lay sleeping downstream.<br> It was a race they were about to lose.</p>
+      <p class="overflow_text text_two">Just after 1:00 am, the sky cleared and stars emerged from behind the storm clods. There was an eerie calm as someone yelled, "The water level is going down! The flood is retreating!"</p>
+      <p class="overflow_text text_three">There was little chance to enjoy that calm. One survivor recalled that a few seconds later it "sounded like the sky was collapsing and the earth was cracking." The equivalent of 280,000 Olypmic-sized swimming pools burst through the crumbling dam, taking with it entire towns and as many as 171,000 lives.</p>
+    </article>
+    <form>
+      <label for="first_name">First Name</label>
+      <input id="first_name" type="text">
+      <br>
```

Comment:
The only use case for a `<br>` is to break text if needed, like in an address. You should never use it to create breaks in the UI like this.